apps/angular/src/app/features/auth/login/login.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="auth-container">
+	<h1 class="auth-title">Login</h1>
+	<div class="auth-common-errors-container" *ngIf="commonErrors$ | async as errors">
+		<mat-error *ngIf="commonErrors$ | async as error">
+			{{ error }}
+		</mat-error>
+	</div>
+	<form [formGroup]="loginForm" (ngSubmit)="login()" class="auth-form">
+		<mat-form-field color="accent">
+			<mat-label color="accent">Email</mat-label>
+			<input formControlName="email" matInput />
+			<mat-error *ngIf="loginForm.controls.email.hasError('required') && loginForm.controls['email'].touched">
+				{{ "Email" | errorMessage : "required" }}
+			</mat-error>
+			<mat-error *ngIf="loginForm.controls.email.hasError('email') && loginForm.controls['email'].touched">
+				{{ "Email" | errorMessage : "email" }}
+			</mat-error>
+			<mat-error *ngFor="let error of loginForm.controls.email.getError('invalid')">
+				{{ error }}
+			</mat-error>
+		</mat-form-field>
+		<mat-form-field color="accent">
+			<mat-label color="accent">Password</mat-label>
+			<input type="password" formControlName="password" matInput />
+			<mat-error *ngIf="loginForm.controls.password.hasError('required') && loginForm.controls.password.touched">
+				{{ "Password" | errorMessage : "required" }}
+			</mat-error>
+			<mat-error *ngIf="loginForm.controls.password.hasError('minlength') && loginForm.controls.password.touched">
+				{{ "Password" | errorMessage : "minlength" }}
+			</mat-error>
+			<mat-error *ngFor="let error of loginForm.controls.password.getError('invalid')">
+				{{ error }}
+			</mat-error>
+		</mat-form-field>
+		<button class="auth-submit" type="submit" mat-raised-button color="primary">Login</button>
+	</form>
+	<a routerLink="/auth/register" class="auth-link" href="">Do not have an account yet?</a>
```

Comment:
There is no need to provide `href`
```suggestion
	<a routerLink="/auth/register" class="auth-link">Do not have an account yet?</a>
```